fdbclient/ServerKnobs.cpp

```diff
@@ -644,6 +644,8 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( MAX_STORAGE_COMMIT_TIME,                             120.0 ); //The max fsync stall time on the storage server and tlog before marking a disk as failed
 	init( RANGESTREAM_LIMIT_BYTES,                               2e6 ); if( randomize && BUGGIFY ) RANGESTREAM_LIMIT_BYTES = 1;
 	init( ENABLE_CLEAR_RANGE_EAGER_READS,                       true );
+	init( QUICK_GET_VALUE_FALLBACK,                             true );
```

Comment:
Can SRE toggle the knob online?
If so, can we BUGGIFY it 
`if( randomize && BUGGIFY ) QUICK_GET_VALUE_FALLBACK  = false;`

Reply:
Yes SRE can toggle this knob online. But the tests (which doesn't do application level fallback) is expected fail when it needs fallback. This is a contingency plan if upper layer doesn't like the performance of fallback, so they can fallback themselves using the old API.

Reply:
Can we check changing it to false won't break existing tests?

Reply:
Yes, it won't break existing tests. This knob is in a codepath where other requests won't hit.

Reply:
I think we should BUGGIFY it to make sure. 
We can disable those tests that is expected to fail when the knob is set to false. 
To disable such test, you can simply return success when knob is false.